# MeasEpochChannelType2 block
   
uint8  type
uint8  lock_time
uint8  cn0
uint8  offsets_msb
int8   carrier_msb
uint8  obs_info
uint16 code_offset_lsb
uint16 carrier_lsb
uint16 doppler_offset_lsb